diff --git a/cmake/DakotaFindSystemTPLs.cmake b/cmake/DakotaFindSystemTPLs.cmake
--- a/cmake/DakotaFindSystemTPLs.cmake
+++ b/cmake/DakotaFindSystemTPLs.cmake
@@ -12,7 +12,7 @@ macro(dakota_find_boost)
 
   # Dakota requires the specified compiled Boost library components
   # Dakota requires Boost 1.70 or newer ; enforce for all libs in the build
-  set(dakota_boost_libs program_options regex serialization system)
+  set(dakota_boost_libs program_options regex serialization)
 
   if(DAKOTA_APPLE_FIX_BOOSTLIBS)
     # This approach requires separate include and lib dirs
@@ -38,8 +38,8 @@ macro(dakota_find_boost)
   find_package(Boost 1.70 REQUIRED COMPONENTS ${dakota_boost_libs})
   #message(STATUS "Found Boost version ${Boost_VERSION}")
 
-  set(DAKOTA_BOOST_TARGETS Boost::boost Boost::program_options
-    Boost::regex Boost::serialization Boost::system)
+  set(DAKOTA_BOOST_TARGETS Boost::boost Boost::program_options Boost::regex
+    Boost::serialization)
 
   set(dakota_boost_libs "${dakota_boost_libs}" CACHE STRING "")
   set(dakota_boost_version "${Boost_VERSION_MAJOR}.${Boost_VERSION_MINOR}.${Boost_VERSION_PATCH}" CACHE STRING "")
diff --git a/src/plugins/CMakeLists.txt b/src/plugins/CMakeLists.txt
--- a/src/plugins/CMakeLists.txt
+++ b/src/plugins/CMakeLists.txt
@@ -17,15 +17,13 @@ set_target_properties(generic_python_plugin PROPERTIES CXX_STANDARD 17)
 set_target_properties(generic_python_plugin PROPERTIES CXX_VISIBILITY_PRESET hidden)

-find_package(Boost 1.70 REQUIRED COMPONENTS system)
 # handle special case for gcc filesystem support
 if((CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0) OR DAKOTA_LINK_STDCPPFS)
   target_link_libraries(generic_python_plugin
-    PUBLIC Boost::system stdc++fs
+    PUBLIC stdc++fs
     PRIVATE pybind11::embed Python::Python
   )
 else()
     target_link_libraries(generic_python_plugin
-      PUBLIC Boost::system
       PRIVATE pybind11::embed Python::Python
     )
 endif()
diff --git a/src/surrogates/unit/CMakeLists.txt b/src/surrogates/unit/CMakeLists.txt
--- a/src/surrogates/unit/CMakeLists.txt
+++ b/src/surrogates/unit/CMakeLists.txt
@@ -25,7 +25,7 @@ endif()

 dakota_add_unit_test(NAME surrogates_polynomial_regression
   SOURCES PolynomialRegressionTest.cpp
-  LINK_LIBS dakota_surrogates Boost::system )
+  LINK_LIBS dakota_surrogates )

 dakota_add_unit_test(NAME surrogates_tools
   SOURCES SurrogatesToolsTest.cpp
@@ -55,7 +55,7 @@ if(DAKOTA_PYTHON_SURROGATES)

   dakota_add_unit_test(NAME surrogates_python_pybind11
     SOURCES PythonSurrogatesTest.cpp
-    LINK_LIBS dakota_surrogates Boost::system )
+    LINK_LIBS dakota_surrogates )
   # Rationale: This includes Teuchos headers and needs to link to the
   # ParameterList components. It also includes SurrogatesPython.hpp, which
   # depends on pybind11 headers that are build-only details of dakota_surrogates.
